src/transformations/RenameEntitiesTransformation.cpp

```diff
@@ -166,6 +177,8 @@ ITransformation * RenameEntitiesTransformation::buildFromConfig(const YAML::Node
     const auto seed = config["seed"].as<int>();
     const auto max_tokens = config["max tokens"].as<int>();
     const auto max_token_len = config["max token len"].as<int>();
+    const auto hash_token = config["hash token"] != nullptr ? config["hash token"].as<string>() : "d";
+    const auto with_hash = config["with hash"] != nullptr && config["with hash"].as<bool>();
```

Comment:
Let's rewrite this as follows to make it more readble:
```
const auto with_hash =  (config["with hash"] != nullptr) ? config["with hash"].as<bool>() : false;
```
Also as a very nervous person, I would suggest using brackets both in this and in the previous line to avoid unexpected behavior due to operations priority